### DISCUSSION TITLE
Toga DetailedList Accessors and flex style

### DISCUSSION THREAD

**Discussion by Eric35470** [beeware/toga#2234] [Debugging help]
Hello, it's my first publication, here. Sorry, if I don't respect the code.
I want to use a DetailedList with toga and change the accessors.
I obtain an error: **'str' object has no attribute '_impl'** even if I keep "icon", "title", "subtitle".
I am using a windows platform and I want to vreate an Android application.
On other hand, I would like to have a maximum eight of my list using flex parameter of Pack. But, it doesn't work.
Do you know why ?
Thanks in advance for your help.
Eric

**Comment by freakboy3742** [comment]
It's very difficult to comment on why something isn't working when you haven't provided any details on what it is you've *done*. An abstract description and one line of an error message doesn't give us enough detail to reproduce the problem or see where you might have gone wrong.

We'd be happy to help, but you'll need to share the code that you're using - in this case, at the very least the way you're constructing your DetailedList.

**Reply by Eric35470** [reply]
Thanks for your answer.
You are right, I solve myself the issue on accessors, I used it stating by the 'icon' one, instead the 'tiitle' one.
I used  "icon", "title", "subtitle" unstead of  "title", "subtitle"n "icon".
So I fixed the accessors issue.

**Reply by Eric35470** [reply]
About the flex parameter, it's also fixed. I forgot to define the box which contents my DetailedLis as Flex.
Sorry for the spam.
